doc/integrator/docker.rst

```diff
@@ -72,6 +72,17 @@ This will remove::
 * Images with version on name as `<none>`.
 * Unnamed dangling volumes.
 
+To remove unused volumes created by possibly unconsistent volume declarations
+in the ``Dockerfile`` compared to ``docker-compose.yml``:
+
+.. prompt:: bash
+
+    #!/bin/bash
+    for vol in $(docker volume ls | awk '{print $2}' | grep -v VOLUME)
+    do
+      docker volume rm $vol
+    done
+
```

Comment:
For last release of Docker you can use::
`docker system prune --volumes`
or::
`docker system prune --all`